src/Oro/Bundle/DataGridBundle/Resources/doc/backend/extensions/formatter.md

```diff
@@ -12,6 +12,7 @@ column_name:
     type: field # default value `field`, so this key could be skipped here
     frontend_type: date|datetime|decimal|integer|percent|currency|select|text|html|boolean # optional default string
     data_name: someAlias.someField # optional, key in result that should represent this field
+    divisor: some number # optional if you need to divide a numeric value by a number before rendering it
```

Comment:
In https://github.com/orocrm/platform/pull/704/files#diff-ec92bcd45dd3ddbb7ca0081d1af0c23eR43 you use `100` as a default value, can you provide a quick example here as what kind of divisor value make sense ?